src/tiled/mapitem.cpp

```diff
@@ -357,7 +357,24 @@ void MapItem::repaintRegion(const QRegion &region, TileLayer *tileLayer)
 
 void MapItem::documentChanged(const ChangeEvent &change)
 {
+    // todo: reload hides the darkening of all but currently selected map
```

Comment:
Alright, that's just the "selected layers highlight" disappearing, because currently after reload no layers are selected.